static/styles/left_sidebar.css

```diff
@@ -235,7 +235,7 @@ li.active-sub-filter {
 
     .count {
         margin-right: 20px;
-        margin-top: 2px;
+        margin-top: 0;
```

Comment:
This change applies to all of the count elements in the top-left region.  I'm worried that this change breaks the vertical alignment of e.g. the unread counts for private messages.    (I think I see that in your screenshot, though it's hard to tell since they aren't hovered).

Can you figure out a different fix?

Reply:
@timabbott I had tried removing the margin top and then trying that border should not touch the bottom of that blue box, but without it i am finding it difficult.